.github/workflows/deps.yml

```diff
@@ -8,27 +8,14 @@ on:
         branches: [master]
         paths: [Cargo.lock]
 
-env:
-    RUSTFLAGS: -D warnings
-    CARGO_TERM_COLOR: always
-
 concurrency: deps-${{ github.head_ref || github.run_id }}
 
 jobs:
     deny:
-        name: deny (${{ matrix.checks }})
+        name: deny
```

Comment:
right, we def don't need a matrix here if we use `check all` which does both

Reply:
A thing I noticed about this is that if `check advisories` fails with an error, it will continue regardless, but then if the next matrix step succeeds the entire job succeeds, silencing the advisory..

Reply:
I initially took it from [reth](https://github.com/paradigmxyz/reth/blob/main/.github/workflows/deny.yml), should I update it there too?

Reply:
please do, ty!